src/app/components/planner-list/planner-list.component.less

```diff
@@ -104,7 +104,7 @@
   .toggle-children { display: none !important; }
 }
 .tree-list-item {
-  display: inherit;
+  display: inline-flex;
```

Comment:
Planner is using tree list component but styles are not there in tree list component